backend/api/Database/Context/InitDb.cs

```diff
@@ -87,7 +87,7 @@ private static List<Mission> GetMissions()
             EchoMissionId = 95,
             IsarMissionId = "1",
             MissionStatus = MissionStatus.Pending,
-            StartTime = DateTimeOffset.UtcNow.AddHours(3),
+            StartTime = DateTimeOffset.UtcNow,
```

Comment:
Having these three missions available like this might be annoying. Every time we start Flotilla locally it will immediately try to schedule a mission. 

Are they used for testing in any way? We can also have missions that are not `Pending` but marked as successful etc. to show up in the other tables. For `Pending` maybe having none is better but check if there are tests relying on them.

Reply:
There are tests relying on them. Suggest that modification of these placeholder missions get made into a separate issue

Reply:
The missions have been changed to successful such that they don't start